### ISSUE TITLE
Failure to upload reported for iOS update 14.8

### ISSUE THREAD

**Issue by DavidRoy** [NERC-CEH/irecord-butterflies-app#103]
Reported on 15/9/21 by one of the most prolific users of the app = recent iOS update to v14.8 has stopped the iRecord butterflies uploads.

I've asked for phone version as I can still upload ok.

@RichardFoxBC - can you confirm you can still upload ok

**Comment by kazlauskis** [comment]
I've looked this up and it seems the Indicia API module finds the request unauthenticated. Which essentially means that for some reason either the app isn't correctly attaching the user credentials, the Indicia API module isn't recognising them or the user has changed email/password on the website and this has diverged from the ones in the app.

For now my suggestion would be to logout and login again - unchecking the 'Discard local data' when logging out. Or changing the email and password back to the original ones if these were changed recently. 

When the iRecord site is upgraded to DP9 then this will use entirely different authentication which should fix the problem of the user changing their account password.
